.github/workflows/stale-repos.yml

```diff
@@ -1,19 +1,47 @@
 on:
   schedule:
-  - cron: "0 */6 * * *" # every 6 hours
+    - cron: "0 */6 * * *" # every 6 hours
+  pull_request:
+    paths:
+      - 'configs/*.json'
+
+name: Check Stale Repos
 
 jobs:
   matrix:
     name: Find stale repositories
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - id: repos
+        name: find repositories to check
```

Comment:
This step should only run on `pull_request`, right?
We still need to check all repos on `cron`.